aten/src/TH/THGenerateHalfType.h

```diff
@@ -3,7 +3,7 @@
 #endif
 
 #include <TH/THHalf.h>
-#define scalar_t THHalf
+#define scalar_t at::Half
```

Comment:
curious: why'd you make this change?

Reply:
I remember getting compile errors with THHalf but can't remember what they were

Reply:
The current version of the code compiles without this change... reverted.